cartography/models/common/helpers.py

```diff
@@ -0,0 +1,74 @@
+import logging
+from typing import Any
+from typing import Dict
+from typing import List
+from typing import Type
+
+import neo4j
+
+from cartography.client.core.tx import load
+from cartography.models.common.human import HumanSchema
+from cartography.models.core.nodes import CartographyNodeSchema
+
+logger = logging.getLogger(__name__)
+
+
+def _load_abstracted_nodes(
+    neo4j_session: neo4j.Session,
+    schema: Type[CartographyNodeSchema],
+    update_tag: int,
+    data: List[Dict[str, Any]],
+    mapping: Dict[str, str],
+    id_field: str = "id",
+):
```

Comment:
Add a comment about how this function standardizes the ways that an email address can be referred to